.github/workflows/clang-analysis.yml

```diff
@@ -0,0 +1,50 @@
+name: "Code analyzis (clang) - action"
+
+on:
+  push:
+    branches:
+    - 'master'
+    - 'dev'
+    - 'stable'
+  pull_request:
+    branches:
+    - master
+    - dev
+    - 'release-*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+          submodules: recursive
+
+    - name: Update package database
+      run: sudo apt --assume-yes update
+
+    - name: Install git
+      run: sudo apt --assume-yes install git
+
+    - name: Install python
+      run: sudo apt --assume-yes install python3-wheel python3-setuptools
+
+    - name: Install meson ninja
+      run: pip3 install --user meson ninja
+
+    - name: Install clang
+      run: sudo apt --assume-yes install clang clang-tools
+
+    - name: Analysis
+      run: |
+        export PATH=${HOME}/Library/Python/3.8/bin:${HOME}/Library/Python/3.9/bin:${HOME}/.local/bin:${PATH}
+        scan-build -sarif -o reports "meson build && ninja -C build"
+
+    - name: Move reports
+      run: mv reports/*/*.sarif reports/
+
+    - name: Upload SARIF file
+      uses: github/codeql-action/upload-sarif@v1
```

Comment:
where is it possible to see the result?

Reply:
Location:
![simplescreenrecorder-2020-12-16_22 26 40](https://user-images.githubusercontent.com/32521367/102408410-d0ee2880-3fed-11eb-82d7-a8c06a615ae9.gif)

TLDR: The report isn't available for Rizin because it wasn't run on a base branch.
![2020-12-16-222551_651x314_scrot](https://user-images.githubusercontent.com/32521367/102408486-eebb8d80-3fed-11eb-853b-2521ef94a0d0.png)